website/src/arcade/examples.ts

```diff
@@ -63,4 +63,77 @@ export const EXAMPLES = {
         })
     `),
   },
+
+  "Multiple root queries": {
+    dataset: "pokemon",
+    code: wrapStandardQuery(`
+      q("")
+        .grab({
+          numPokemon: ["count(*[_type == 'pokemon'])", q.number()],
+          allTypeNames: q("*").filterByType("poketype").grabOne("name", q.string()),
+        })
+    `),
+  },
+
+  "Raw GROQ Functions": {
+    dataset: "pokemon",
+    code: wrapStandardQuery(`
+      q("*")
+        .filterByType("pokemon")
+        .slice(0, 8)
+        .grab({
+          name: q.string(),
+          // pass raw query and a schema 
+          numTypes: ["count(types)", q.number()],
+          foo: ["coalesce(foo, 'not there')", q.string()]
+        })
+    `),
+  },
+
+  "Forking Selection": {
+    dataset: "pokemon",
+    code: wrapStandardQuery(`
+      q("*")
+        .filterByType("pokemon")
+        .filter("name in ['Bulbasaur', 'Charmander']")
+        .select({
+          // For Bulbasaur, grab the HP
+          'name == "Bulbasaur"': {
+            _id: q.string(),
+            name: q.literal("Bulbasaur"),
+            hp: ["base.HP", q.number()],
+          },
+          // For Charmander, grab the Attack
+          'name == "Charmander"': {
+            _id: q.string(),
+            name: q.literal("Charmander"),
+            attack: ["base.Attack", q.number()],
+          },
+          // For all other pokemon, cast them into an unsupported selection
```

Comment:
should the reader know what an unsupported selection is here?

Reply:
idk, I just stole this from our docs 😆 I'll add a little note